cmd/plasmacli/store/keystore.go

```diff
@@ -18,6 +18,8 @@ const (
 
 	NewPassphrasePrompt       = "Enter new passphrase for your key:"
 	NewPassphrasePromptRepeat = "Repeat passphrase:"
```

Comment:
we should be using this on export

Reply:
ExportPassphrasePrompt    = "Enter new passphrase for exported key:"
- We are using this right now -- should I delete ExportPassphrasePrompt and use NewPassphrasePrompt instead?